content/en/docs/instrumentation/python/benchmarks.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Benchmarks
+weight: 101
+cSpell:ignore: Elems rrggbbaa
+---
+
+<link rel="stylesheet" href="/css/benchmarks.css">
+
+The OpenTelemetry Python SDK runs benchmark tests on every commit to the
+[opentelemetry-python](https://github.com/open-telemetry/opentelemetry-python/)
+repository. The intent of these tests is to track performance trend of critical
+operations over time. These test are not a replacement for end-to-end
+performance testing.
+
+  <div class="container">
+    <main id="main"></main>
+  </div>
+
+  <footer>
+    <button id="dl-button">Download data as JSON</button>
+    <div class="spacer"></div>
+    <div class="small">Powered by <a rel="noopener"
+        href="https://github.com/marketplace/actions/continuous-benchmark">github-action-benchmark</a></div>
+  </footer>
+
+  <script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.2/dist/Chart.min.js"></script>
+  <script src="https://open-telemetry.github.io/opentelemetry-python/benchmarks/data.js"></script>
+  <script id="main-script">
```

Comment:
I haven't looked at the other benchmark files, but is the JS here the same?

Reply:
I think so?

[Python](https://open-telemetry.github.io/opentelemetry-js/benchmarks/data.js)
[Js](https://open-telemetry.github.io/opentelemetry-js/benchmarks/data.js